extensions/jsdbg/core/jsdbg.js

```diff
@@ -446,14 +446,18 @@ Loader.OnLoad(function () {
                 {name:"module", type:"string", description:"The module containing the symbol."},
                 {name:"symbol", type:"string", description:"The symbol to evaluate."},
                 {name:"typeName", type:"string", description: "The type name of the symbol to look up."},
+                {name:"namespace", type:"string", description: "The namespace the symbol is in. Required on some platforms."},
                 {name:"callback", type:"function(object)", description:"A callback that is called when the operation succeeds or fails."}
             ]
         },
-        LookupGlobalSymbol: function(module, symbol, typeName, callback) {
+        LookupGlobalSymbol: function(module, symbol, typeName, namespace, callback) {
+            var namespaceStr = "";
+            if (namespace)
```

Comment:
Consider applying the same pattern to typename. This feels cleaner than having multiple calls to JsonRequest.

Reply:
Done